superset/dashboards/filter_state/commands/create.py

```diff
@@ -23,13 +25,17 @@
 
 
 class CreateFilterStateCommand(CreateKeyValueCommand):
-    def create(self, cmd_params: CommandParameters) -> bool:
+    def create(self, cmd_params: CommandParameters) -> str:
         resource_id = cmd_params.resource_id
         actor = cmd_params.actor
-        key = cache_key(resource_id, cmd_params.key)
+        contextual_key = cache_key(actor.get_user_id(), resource_id)
+        key = cache_manager.filter_state_cache.get(contextual_key)
+        if not key:
+            key = token_urlsafe(48)
```

Comment:
Not introduced here, but noticed this only now.. Since we're using the same `token_urlsafe(48)` both in dashboard and explore form data commands, could we DRY this by creating a util under the `key_value` package that hides both the token generator and the magic number? Something like

```python
def get_key() -> str:
    return token_urlsafe(48)
```

Reply:
Nice catch! Done.